extension/chrome/elements/compose-modules/compose-render-module.ts

```diff
@@ -24,18 +24,20 @@ import { KeyUtil } from '../../../js/common/core/crypto/key.js';
 
 export class ComposeRenderModule extends ViewModule<ComposeView> {
 
-  public renderReplyMsgComposeTable = async (method: 'forward' | 'reply' = 'reply'): Promise<void> => {
+  private replyBoxMethod!: 'reply' | 'forward';
```

Comment:
We could consider `responseMethod` or maybe `responseReachIntention`.. just trying to think

Reply:
`responseMethod` look good 👍